### PR TITLE
git collaboration workflow instructions

### PR DESCRIPTION
**Pull request by paddymul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -565,3 +565,73 @@ const results = await testSheet.testRecords(inputRows)
 expect(results).toMatchObject(expectedOutputRows)
 ```
 
+## Using git and github to collaborate with Flatfile
+
+The platform-sdk allows you to use git to manage your Flatfile config.  You can also use git to get feedback on your implementation from Flatfile engineers.  Using git can be tricky, we recommend you follow the prescribed pattern to securely share your code with Flatfile.  When you share code in this way Flatfile can much more quickly diagnose problems you are having and provide guidance.  Flatfile greatly values interaction like this because it helps us shape the future of the platform-sdk and other features.  We see your code, admire your ingenuity, and see where you're struggling then we shape the platform-sdk to ease those rough spots
+
+Steps
+- Clone the platform-sdk-starter
+` $ git clone git@github.com:FlatFilers/platform-sdk-starter.git`
```
Comment (hansjhoffman) [inline README.md:574]: 🔧 Commands with extraneous info like `$/#` I've found always end up tripping up beginners b/c they will copy-and-paste and the ask "why is this not working?"

**Comment by bangarang** [comment]
@paddymul Maybe we should just add the screenshots in here to the Readme instead of as separate failes?

File: README.md (hunk 1 above)
Comment (paddymul) [reply README.md:574]: That is annoying for copy/paste.  
Between `shell` `bash` and `console` do you have a reccomendation?

https://stackoverflow.com/questions/20303826/how-to-highlight-bash-shell-commands-in-markdown

maybe only show the prompt in screenshots... Show the whole flow in one screenshot?

Comment (hansjhoffman) [reply README.md:574]: My recommendation would be to use `bash` and just leave off any console related things -- commands only. Screenshots are obviously hard to "copy and paste" into my terminal (I'm lazy sometimes lol).

**Comment by paddymul** [comment]
> 

I'm not sure what you mean

File: README.md [hunk 2]
```diff
@@ -565,3 +565,75 @@ const results = await testSheet.testRecords(inputRows)
...
+```bash
+git clone git@github.com:FlatFilers/platform-sdk-starter.git
+```
+- Create your own private repo
+Navigate to https://github.com/new
+Fill out details as in the picture, take care to make the repo private and not check any other boxes.  The example repo name for this tutorial will be `private-platform-sdk-starter` and it lives at https://github.com/paddymul/private-platform-sdk-starter (check that your new repo isn't accessible from a non-logged in private browsing tab before proceeding)
+
+![Create a private repo](/assets/GH-create-private-repo.png)
+- Push to your new private repo
+```
```
Comment (hansjhoffman) [inline README.md:583]: ⛳ I think you could add `bash` here even with the comments below and it wont complain

File: README.md [hunk 3]
```diff
@@ -565,3 +565,75 @@ const results = await testSheet.testRecords(inputRows)
 expect(results).toMatchObject(expectedOutputRows)
 ```
 
+## Using git and github to collaborate with Flatfile
```
Comment (hansjhoffman) [inline README.md:568]: ⛳ github -> GitHub

File: README.md [hunk 4]
```diff
@@ -565,3 +565,75 @@ const results = await testSheet.testRecords(inputRows)
...
+
+https://github.com/paddymul/private-platform-sdk-starter/settings
+Then navigate to the access pane 
+https://github.com/paddymul/private-platform-sdk-starter/settings/access
+at this point github will ask for two factor auth, fill it out.
+![Add a collaborator](/assets/GH-Add-a-collaborator-Rock.png)
+
+Add the names of `bangarang` and `paddymul`
+
+- Notify Flatfile that you shared the PR with us
```
Comment (hansjhoffman) [inline README.md:634]: ❔ How should they notify FF (outside of a GitHub notification)? Email Support? Not everyone will have their own implementation team

**Review by hansjhoffman — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (elisadinsmore) [reply README.md:634]: > Once you've added collaborators, please notify Flatfile Support. If you have an ongoing ticket with our Success or Support teams, you can update the ticket to let them know that you have shared your PR with our team. If you do not yet have an open ticket, email support@flatfile.com with a description of your problem and include a note to let us know that you have already shared your PR.

**Pull request closed without merging**